### PR TITLE
Updated typos and clarified instructions

### PR DESCRIPTION
**Pull request by shaeHansen**
Corrected a few typos and clarified a few areas in the instructions.

### CODE REVIEW

File: docs/qiita-16S-analysis.rst [hunk 1]
```diff
@@ -169,7 +169,7 @@ Faith's Phylogenetic Diversity Index
 
 The final alpha diversity analysis in this tutorial uses Faith’s phylogenetic diversity index. This index also measured abundance and diversity but considers the phylogenetic distance spanning all features in a sample. The results can also be displayed as a phylogeny, rather than as a plot.
 
-To perform an alpha diversity analysis using Faith's phylogenetic diversity index, select the "rarefid table (BIOM)" artifact in the processing network and select "Process". Select "Alpha diversity (phylogenetic)" from the drop-down menu. The parameters will appear below the workflow diagram:
+To perform an alpha diversity analysis using Faith's phylogenetic diversity index, select the "rarefied table (BIOM)" artifact in the processing network and select "Process". Select "Alpha diversity (phylogenetic)" with the Qiime2 command of "alpha phylogenetic" not "alpha phylogenetic old" from the drop-down menu. The parameters will appear below the workflow diagram:
```
Comment (antgonza) [inline docs/qiita-16S-analysis.rst:172]: Is this right? Where did you see  `alpha phylogenetic old`?

**Review by antgonza — COMMENTED** [review]
Looks good, one question! Thank you @shaeHansen

File: docs/qiita-16S-analysis.rst (hunk 1 above)
Comment (shaeHansen) [reply docs/qiita-16S-analysis.rst:172]: image
When I click on the rarefied table and select process, here's a screenshot with the alpha phylogenetic old highlighted.  Should it not be there?

Comment (antgonza) [reply docs/qiita-16S-analysis.rst:172]: Got it, thank you for clarifying!

**Pull request merged**